src/runtime/mod.rs

```diff
@@ -13,7 +13,10 @@ thread_local! {
     pub(crate) static CONTEXT: RuntimeContext = RuntimeContext::new();
 }
 
-/// The Runtime executor
+/// The Runtime Executor
+///
+/// io_uring instances are always single-threaded. Correspondingly, this runtime will
```

Comment:
@Noah-Kennedy Is this correct? Is it over stating that `io-uring` instances are always single-threaded? I just got the impression all its calls are synchronous. And the uring interface in the kernel isn't single-threaded. In fact there is at least one parameter we can set when we know it will be single threaded as an optimization.

Reply:
As far as I can tell, currently the only interaction with the Tokio Uring Driver is inside of `Runtime::block_on` and it does this via `tokio::runtime::LocalSet` which is the !Send current-thread set of tasks.

Reply:
This runtime is single-threaded mostly to make this as simple as possible.

Reply:
Anyway, the comment on line 18 is wrong. The io_uring instances are *not* always single-threaded. How the io_uring crate is used is very dependent on the opinions of the crate wants to  use it.

Reply:
Ok.  I have removed that line and restructured a bit.